### DISCUSSION TITLE
Font options for LineEdit?

### DISCUSSION THREAD

**Discussion by LouisGariepy** [slint-ui/slint#434] [Q&A] [answered]
Hello! First of all, thanks for this wonderful project, I am falling in love with it :smile: .

I was wondering if there is a way to change the font size for this widget. Since there are width/height properties, it is natural to assume that there is a property to change the font size, but that doesn't seem to be the case.

Thanks!

**Comment by LouisGariepy** [comment, marked as answer]
I took a bit more time to inspect the source and now I feel confident I can do exactly that by creating my own LineEdit widget.

**Reply by tronical** [reply]
You have a point though, maybe this property should be added. If `Text and `TextInput` have `font-size`, then `LineEdit` should have it, too IMO :). I'll create an issue from this.
